### DISCUSSION TITLE
Unable to start playwright tests

### DISCUSSION THREAD

**Discussion by hananurrehman** [inventree/InvenTree#8507] [General]
I'm trying to run the playwright tests but running into this issue:
```
(venv) vscode ➜ /home/inventree/src/frontend (master) $ npx playwright test --ui
Error: Failed to launch: Error: spawn /home/vscode/.cache/ms-playwright/chromium-1134/chrome-linux/chrome ENOENT
    at ChildProcess.<anonymous> (/home/inventree/src/frontend/node_modules/playwright-core/lib/utils/processLauncher.js:132:14)
    at Object.onceWrapper (node:events:632:26)
    at ChildProcess.emit (node:events:529:35)
    at ChildProcess._handle.onexit (node:internal/child_process:290:12)
    at onErrorNT (node:internal/child_process:477:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**Comment by hananurrehman** [comment]
I'm running docker through colima.

**Reply by SchrodingersGat** [reply]
Are you running our default docker image? We currently have a limitation where playwright won't run on our docker image (or devcontainer) as playwright is not supported on alpine.

**Reply by hananurrehman** [reply]
I just followed the steps from the docs. So is there a way I can run the playwright tests?

**Reply by hananurrehman** [reply]
Since your project is open source, I wanted to contribute to your UI tests but it seems the setup is proving a bit too complex for me right now.
@SchrodingersGat would it be ok with you if I used your demo site to write some playwright tests as part of my portfolio?

**Reply by SchrodingersGat** [reply]
Currently the playwright tests only work if you have the project  setup outside of docker (my setup for e.g. is running on WSL).

**Reply by hananurrehman** [reply]
would it be ok with you if I used your demo site to write some playwright tests as part of my portfolio?

**Reply by SchrodingersGat** [reply]
@hananurrehman I'm not sure 100% what you mean here. Would you contribute the tests back to the project?

**Reply by hananurrehman** [reply]
My initial plan was to contribute to your project by writing playwright tests, the purpose being to showcase my skills on my github profile.

However, since the project setup is a bit complicated, I'm thinking of just creating a separate repo that houses playwright tests while using your demo site as the target. 

For contributing back to your project repo, I will keep that on my to-do list once I find the time to set it up properly on my side.
